code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1132,7 +1132,7 @@
 	cell = null
 
 /mob/living/silicon/robot/mouse_buckle_handling(mob/living/M, mob/living/user)
-	if(can_buckle && istype(M) && !(M in buckled_mobs) && ((user!=src)||(a_intent != INTENT_HARM)))
+	if(user.can_buckle && istype(M) && !(M in buckled_mobs) && ((user!=src)||(a_intent != INTENT_HARM)))
```

Comment:
`can_buckle` is a var for controlling if things can be buckled to that movable, not for determining if that thing is allowed to buckle other things to eachother.

You need to implement a check yourself for this kind of thing

Reply:
In this case filling the missing istype should be enough.